src/arrayMethodShift.js

```diff
@@ -5,7 +5,19 @@
  */
 function applyCustomShift() {
   [].__proto__.shift2 = function() {
-    // write code here
+    if (this.length === 0) {
+      return undefined;
+    }
+
+    const firstLetter = this[0];
```

Comment:
это может быть что угодно, не обязательно буква
```suggestion
    const firstElement = this[0];
```